brainstormforce/astra
# AST-2913 - Fix: Extra Spacing in Customizer Sections

### Description

- Need to removed single post design section extra spacing when we enable related post
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2913

### Screenshots

[Annotation on 2023-01-31 at 09-21-05.png](https://d.pr/i/gijlPU) 
Same issue found in product catalog design section - [Annotation on 2023-02-01 at 16-35-07.png](https://d.pr/i/Kxr1Wy)

### Types of changes




Fix: Extra Spacing in Customizer Sections

### How has this been tested?

- Check Blog > Single Post.
- Enable Related Posts, and check Design Tab.
- The Related Post section should have margin-top as 16px, not 32px.
- Please check same for WooCommerce Product Catalogue > Shop Card Styling in Design Tab after disabling Addon.

### Checklist:
